lib/transfer.c

```diff
@@ -487,6 +487,12 @@ CURLcode Curl_readrewind(struct connectdata *conn)
 static int data_pending(const struct Curl_easy *data)
 {
   struct connectdata *conn = data->conn;
+
+#ifdef ENABLE_QUIC
+  if(conn->transport == TRNSPRT_QUIC)
+    return Curl_quic_data_pending(data);
```

Comment:
I notice that quic_data_pending always returns false for quiche. There is no need to check ssl-data-pending here?

Reply:
When we use QUIC we don't really use TLS by any means or methods we otherwise use in libcurl. We just do QUIC, so no we don't need that then.